analysis_options.yaml

```
@@ -21,8 +21,11 @@ linter:
   # `// ignore_for_file: name_of_lint` syntax on the line or in the file
   # producing the lint.
   rules:
+    formatter:
+      trailing_commas: preserve
+
     # avoid_print: false  # Uncomment to disable the `avoid_print` rule
     # prefer_single_quotes: true  # Uncomment to enable the `prefer_single_quotes` rule
 
 # Additional information about this file can be found at
-# https://dart.dev/guides/language/analysis-options
+# https://dart.dev/guides/language/analysis-options
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)